### ISSUE TITLE
[Bug] Error message appears many times after invalid login attempt

### ISSUE THREAD

**Issue by lealakkis** [gfdb/soen390#88]
Steps to Reproduce
Try to login twice or more with invalid credential 

Actual
The error message appears many times

Expected
The error message should appear only once no matter the amount of time invalid login is tried

Additional Comments

**Comment by davrine** [comment]
fixed with new pr #126
